tests/test_enhancements.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+def test_get_by_newest():
+    item_a = Clothing(age=2.0)
+    item_b = Decor(age=2.0)
+    item_c = Clothing(age=4.0)
+    item_d = Decor(age=5.0)
+    item_e = Clothing(age=3.0)
+    tai = Vendor(
+        inventory=[item_a, item_b, item_c, item_d, item_e]
+    )
+
+    newest_item = tai.get_by_newest()
+
+    assert newest_item.category == "Clothing"
+    assert newest_item.age == pytest.approx(2.0)
+
+def test_get_by_newest_some_items_with_None_age():
```

Comment:
This test doesn't really prove to me that this is ignoring the None values. Again, from this, it could just be that the first item is being returned immediately and the items with `None` for their age aren't even being examined. We could make this a more robust test by moving the newest item so that it came between to `None` records (it would then be reasonable that the code had to skip over at least _one_ of them). Or have a test where we try to get the newest item from a list containing _only_ items with `None` ages, which should give us `None` as a result.